optaplanner-wb-screens/optaplanner-wb-guided-rule-editor/optaplanner-wb-guided-rule-editor-backend/src/test/java/org/optaplanner/workbench/screens/guidedrule/backend/server/plugin/MultiConstraintMatchPersistenceExtensionTest.java

```diff
@@ -80,6 +99,36 @@ public void unmarshalActionMultiConstraintHardMediumSoftMatch() throws RuleModel
                      actionActionMultiConstraintHardMediumSoftMatch.getActionSoftConstraintMatch().getConstraintMatch());
     }
 
+    @Test
+    public void unmarshalActionMultiConstraintBendableMatchDifferentLength() throws RuleModelDRLPersistenceException {
+        String actionString = "scoreHolder.addMultiConstraintMatch(kcontext, new int[] {-1}, new int[] {2, -3});";
+
+        IAction action = extension.unmarshal(actionString);
+
+        ActionMultiConstraintBendableMatch expectedAction = new ActionMultiConstraintBendableMatch(Arrays.asList(new ActionBendableHardConstraintMatch(0,
+                                                                                                                                                       "-1")),
+                                                                                                   Arrays.asList(new ActionBendableSoftConstraintMatch(0,
+                                                                                                                                                       "2"),
+                                                                                                                 new ActionBendableSoftConstraintMatch(1,
+                                                                                                                                                       "-3")));
```

Comment:
@jomarko Please avoid code overflowing the limit of 120 characters per line. It slightly complicates the reviews on GitHub. Also note that according to the most recent KIE Java Conventions, method call arguments are *not required* to be each on a new line.

Reply:
Understood, will try to fix.